code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -416,16 +422,31 @@
 			to_chat(user, span_warning("The spell had no effect!"))
 		else
 			to_chat(user, span_cultitalic("In a brilliant flash of red, [target] falls to the ground!"))
-			target.Paralyze(16 SECONDS)
+			var/effect_power = 1
+			var/datum/antagonist/cult/user_antag = user.mind.has_antag_datum(/datum/antagonist/cult, TRUE)
+			var/datum/team/cult/cult_team = user_antag.cult_team
+			// If cult ascended, stuns are weaker
+			if(cult_team.cult_ascendent)
+				to_chat(user, span_cultitalic("The cult's energy has thinned out, spreading to [cult_team.size_at_maximum] acolytes. Your spell is weaker!"))
```

Comment:
This should be a bubble message, people won't see it in combat, consider also adding the fact that it weakens in the description